.bumpversion.cfg

```diff
@@ -1,5 +1,19 @@
 [bumpversion]
 current_version = 0.16.0
+commit = True
+# The below regex details an acceptable version number by naming the groups (major, minor, patch, prerel, and
+# prerelversion) and also specifying the valid values for each group (positive integers, `\d+`,  for major, minor,
```

Comment:
comment should just be "integers", not "positive integers".  e.g. 0.16.1

Reply:
Done!